x-pack/plugins/security/server/authentication/index.test.ts

```diff
@@ -358,29 +363,49 @@ describe('setupAuthentication()', () => {
   describe('createAPIKey()', () => {
     let createAPIKey: (
       request: KibanaRequest,
-      body: CreateAPIKeyOptions['body']
+      body: CreateAPIKeyParams
     ) => Promise<CreateAPIKeyResult | null>;
     beforeEach(async () => {
       createAPIKey = (await setupAuthentication(mockSetupAuthenticationParams)).createAPIKey;
     });
 
     it('calls createAPIKey with given arguments', async () => {
-      const { createAPIKey: createAPIKeyMock } = jest.requireMock('./api_keys');
+      const apiKeysInstance = jest.requireMock('./api_keys').APIKeys.mock.instances[0];
       const options = {
         name: 'my-key',
         role_descriptors: {},
         expiration: '1d',
       };
-      createAPIKeyMock.mockResolvedValueOnce({ success: true });
+      apiKeysInstance.create.mockResolvedValueOnce({ success: true });
       await expect(createAPIKey(httpServerMock.createKibanaRequest(), options)).resolves.toEqual({
         success: true,
       });
-      expect(createAPIKeyMock).toHaveBeenCalledWith({
-        body: options,
-        loggers: mockSetupAuthenticationParams.loggers,
-        callAsCurrentUser: mockScopedClusterClient.callAsCurrentUser,
-        isSecurityFeatureDisabled: expect.any(Function),
+      expect(apiKeysInstance.create).toHaveBeenCalledWith(expect.anything(), options);
```

Comment:
__nit:__ actually `expect.anything()` can be replaced with the result of `httpServerMock.createKibanaRequest()` you pass to `createAPIKey` (same below).